game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -22,6 +22,11 @@ object Cell {
    * Ratio of mass lost per second.
    */
   final val MassDecayPerSecond = 0.005f
+
+  /**
+    * Ratio of impact of the mass on the acceleration.
```

Comment:
Could we reword to highlight the fact that this is an exponent (not really a ratio)?

E.g.
`Exponent used on the mass when calculating the acceleration of a cell`